src/types.js

```diff
@@ -19,15 +19,9 @@
  */
 
 /**
- * Props common to Base, abstract components. These are not part of the
- * package API.
+ * Props common to Base components.
```

Comment:
```suggestion
 * Props common to base presentational components.
```

Overlooked this, will fix.